curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5dc17dc8f86c76b9248c6eb5.md

```diff
@@ -40,21 +40,21 @@ const noSpaces = code.replace(/\s/g, '');
 assert(noSpaces.match(/<!--/g).length < 2 && noSpaces.match(/-->/g).length < 2);
 ```
 
-Your comment should contain the text `TODO: Add link to cat photos`.
-
-```js
-assert(code.match(/<!--\s*todo:\s+add\s+link\s+to\s+cat\s+photos\s*-->/i));
-```
-
 Your comment should be above the `p` element. You have them in the wrong order.
 
 ```js
 assert.match(
   code.replace(/\s/g, ''),
-  /<!--todo:addlinktocatphotos--><p>everyonelovescutecatsonline!<\/p>/i
+  /<!--(.*?)--><p>everyonelovescutecatsonline!<\/p>/i
 );
 ```
 
+Your comment should contain the text `TODO: Add link to cat photos`.
+
+```js
+assert.match(code, /<!--\s*todo:\s+add\s+link\s+to\s+cat\s+photos\s*-->/i);
```

Comment:
now the tests are strictly checking for casing and whitespaces, I don't know if we want to remove that. It's in line with other tests of this area of the curriculum if it's less strict tho, so maybe it's a good change